usaspending_api/etl/tests/integration/test_load_multiple_submissions.py

```diff
@@ -159,8 +159,8 @@ def setUp(self):
                     submission_id,
                     updated_at
                 ) (values
-                    (1, 1, '2000-01-01'), (2, 2, '2000-01-02'), (3, 3, '2000-01-03'), (4, 4, '2000-01-04'),
```

Comment:
Is there a reason that this date had to be changed?

Reply:
I added another one with the same original date, it was just later in the list because I wanted to keep the "earlier" one a lower number. The one with an ID of 8 is now what this one was. I needed an entry with more than one publication to test things like null certify and multiple publications.